pages/_document.tsx

```diff
@@ -1,8 +1,10 @@
-import { Html, Head, Main, NextScript } from "next/document";
+import { DocumentProps, Head, Html, Main, NextScript } from "next/document";
+
+export default function Document(props: DocumentProps) {
+  const currentLocale = props.__NEXT_DATA__.props.pageProps.locale ?? "en";
```

Comment:
For the record I don't like this, but it looks like `props.locale` is unset here and doesn't work :thinking: 
```suggestion
  const currentLocale = props.locale;
```

Reply:
I'm pretty sure `props.locale` is reserved for Next's `i18n` feature ._.